MaxText/configs/base.yml

```diff
@@ -194,6 +194,8 @@ prompt: "I love to" # Prompt for language model sampling.
 load_from_prefill_dir: False # If true, decode.py doesn't "prefill" but just reads from directory
 prefill_cache_dir: "" # If set and load_from_prefill_dir, decode.py reads from directory. If set, decode.py writes to directory
 autoregressive_decode_assert: ""
+microbenchmark_prefill_lengths: "64,128,256,512,1024" # The prefill lengths used for microbenchmark
```

Comment:
Can we not we just use regular array notation in the config and load it as a list of ints?
`microbenchmark_prefill_lengths: [64, 128, 256, 512, 1024]`

Reply:
I ran into below error when passing list of ints
```
ValueError: For key 'microbenchmark_prefill_lengths', type <class 'list'> not in dict_keys([<class 'str'>, <class 'int'>, <class 'float'>, <class 'bool'>]), can't pass at the CLI or ENV
```

Reply:
Stupid maxtext thing. You can see how we do this for the sharding rules but actually the comma separated list is much better at the command line and I think it is fine. (I think this is an improvement either way.)